apps/web/lib/slots.ts

```diff
@@ -41,6 +41,13 @@ const getSlots = ({ inviteeDate, frequency, minimumBookingNotice, workingHours,
   const startOfDay = dayjs.utc().startOf("day");
   const startOfInviteeDay = inviteeDate.startOf("day");
   // checks if the start date is in the past
+
+  /**
+   *  TODO: change "day" for "hour" to stop displaying 1 day before today
+   * This is displaying a day as available as sometimes difference between two dates is < 24 hrs.
+   * But when doing timezones an available day for an owner can be 2 days available in other users tz.
+   *
+   * */
```

Comment:
Initially changed day for hours, since it was displaying an empty day as available. Had to revert it since test was failing because of this change. Test needs more investigation.